flow-nodes/api-builder-plugin-format-date/src/flow-nodes.yml

```diff
@@ -11,18 +11,28 @@ flow-nodes:
         parameters:
           date:
             name: Date
-            description: The date to be formatted.
-            required: true
+            description: The date to be formatted (e.g. "2020-08-19"). Defaults to current date/time. For more information, see https://momentjs.com/docs/#/parsing/string/
+            required: false
             initialType: string
             schema:
               type: string
           format:
             name: Format
-            description: The desired format. Supports all Moment.js supported formats.
-            required: true
+            description: The desired format. Defaults to ISO-8601 format. Supports all Moment.js formats. See more valid format examples https://momentjs.com/docs/#/parsing/special-formats/
+            required: false
             initialType: string
             schema:
               type: string
+              default: 'YYYY-MM-DDTHH:mm:ssZ'
+              description: Defaults to ISO-8601 format
```

Comment:
We shouldn't use two descriptions, just the one outside the schema, since one will override the other when we turn it into a flow-node